### ISSUE TITLE
Evaluate TGW.repairMetadata against other changes in elasticity

### ISSUE THREAD

**Issue by keith-turner** [apache/accumulo#3888]
With all of the metadata related changes in elasticity need to evaluate the following function to see if its still relevant and if so what updates it might need.

https://github.com/apache/accumulo/blob/dd619f979aea5ff0c1782fe1d601bb57e47c62f7/server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java#L605

**Comment by keith-turner** [comment]
See the comments on #3872.  Depending on what happens with the repairMetadata function, it may make sense to remove ManagementAction.BAD_STATE and turn it into an error condition.

**Comment by dlmarion** [comment]
@keith-turner - I looked at this function and I think it's still relevant. I have questions about the implementation, but if it has worked in the past I have no issue with it.

**Comment by keith-turner** [comment]
Something that came out of a review of metadata updates in #3858 is that repairmetadata is using a batch writer and not a conditional writer.

**Comment by keith-turner** [comment]
I did research on why the code was added and found that reason it was added is no longer applicable, so decided to remove the code in #4668.  Thinking since the bug it was fixing no longer exists that the code could hide new bugs in elasticity.  We can always add the code back if we find new bugs and that is the best fix for them.  For now would like to see if duplicate locations do ever pop up in elasticity.
